docs/release-source/release/stubs.md

```diff
@@ -226,6 +226,15 @@ Causes the stub to return its <code>this</code> value.
 
 Useful for stubbing jQuery-style fluent APIs.
 
+#### `stub.resolves(obj);`
```

Comment:
This is minor: would you mind renaming the argument to `value`, so that users will understand that it can be any value?